src/main/java/Hello.java

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
To jest ta nasza przykładowa klasa hello world, przeniesiona trochę do innego katalogu